src/main/java/com/parsing/exception/CropImageException.java

```diff
@@ -0,0 +1,8 @@
+package com.parsing.exception;
+
+public class CropImageException extends RuntimeException{
+    public CropImageException(Throwable cause) {
+        super("method \"CropImage.cropImage\" fail. Crop image fail: ", cause);
```

Comment:
move a message to the parameter of the constructor. CropImageException(Throwable cause, String message) and consider using String.format

Reply:
done

Reply:
i have to do it with all exceptions?